src/FlubuCore.WebApi/Controllers/PackagesController.cs

```diff
@@ -8,8 +8,12 @@
 using FlubuCore.WebApi.Model;
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Mvc;
+#if NETCOREAPP3_1
+  using Microsoft.Extensions.Hosting;
+#else
+  using IHostEnvironment = Microsoft.AspNetCore.Hosting.IHostingEnvironment;
```

Comment:
Use the new type name, so that we can simply remove this #if...#else block when we decide to remove support for .NET Core 2.x in the future.

Reply:
Ok, sounds good. The only reason i've added 6.0 milestone to this issue was because. 

Thank you for PR!